src/angularjs/src/app/components/scoremetadata.service.js

```diff
@@ -8,6 +8,47 @@
 (function() {
     'use strict';
 
+    /**
+     Sort metrics in metadata by category, with total first in the category.
+     Add subscoreClass property to metrics that should be indented under the total.
+     */
+    function orderCategories(response) {
```

Comment:
This is ok, and results in the same sort order as explicitly requested by PFB, but it is also brittle in that if we have to add an additional score down the road, or if PFB requests a different ordering, we're no longer respecting the explicit sort order provided by the API that's encapsulated by [AnalysisScoreMetadata.priority](https://github.com/azavea/pfb-network-connectivity/blob/develop/src/django/pfb_analysis/models.py#L631).

A potential alternative to the logic in this method would be to update the AnalysisScoreMetadata.category values in [the fixture](https://github.com/azavea/pfb-network-connectivity/blob/develop/src/django/pfb_analysis/fixtures/analysis-score-metadata.json) to be null for any top level category, and then set the subscore CSS class for any metric that has a non-null category.